extensions/machine-learning/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "machine-learning",
   "displayName": "%displayName%",
   "description": "%description%",
-  "version": "0.8.0",
+  "version": "0.9.0",
   "publisher": "Microsoft",
   "preview": true,
```

Comment:
Remove this line too

Reply:
which line? the extension is still preview. MI is not preview so removed that from the title of the doc in the extension

Reply:
Mistook the point of this PR, nevermind!